tagstudio/src/qt/modals/drop_import.py

```diff
@@ -0,0 +1,82 @@
+from pathlib import Path
+import shutil
+import typing
+
+from PySide6.QtGui import QDropEvent, QDragEnterEvent, QDragMoveEvent
+from PySide6.QtWidgets import QMessageBox
+
+if typing.TYPE_CHECKING:
+    from src.qt.ts_qt import QtDriver
+
+
+def dropEvent(driver: "QtDriver", event: QDropEvent):
+    if not event.mimeData().hasUrls():
+        return
+
+    if not event.mimeData().urls()[0].isLocalFile():
+        return
+
+    urls = event.mimeData().urls()
+    duplicate_filesnames = []
+    for url in urls:
+        if Path(driver.lib.library_dir + "/" + url.fileName()).exists():
```

Comment:
```suggestion
        if Path(driver.lib.library_dir / url.fileName()).exists():
```

Reply:
This will only work once the pathLib PR is done

Reply:
good catch. So it could work like this (suggestion above updated).

Reply:
Yes that should work